### PR TITLE
Updated Vanilla JS getting started guide

### PR DESCRIPTION
**Pull request by renebrandel**
_Issue #, if available:_

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/start/getting-started/vanillajs/data-model.mdx [hunk 1]
```diff
@@ -40,32 +30,37 @@ The CLI should open this GraphQL schema in your text editor.
...
+Now that the API has been successfully created locally, we need to deploy the API to the cloud:
 
 ```bash
 amplify push
 ```
 
 ## Generate code for the GraphQL API
 
-When you run `amplify push`, you will be presented with the option to have all the GraphQL operations found in your schema generated for you in code. Choose the following options:
+When you run `amplify push`, you can generate client code for all the GraphQL operations (create, read, update, delete,l ist). These client code snippets allow you to execute these GraphQL operations more easily with the help of the Amplify JavaScript library. Choose the following options:
```
Comment (josefaidt) [inline src/fragments/start/getting-started/vanillajs/data-model.mdx:63]:
```suggestion
When you run `amplify push`, you can generate client code for all the GraphQL operations (create, read, update, delete, list). These client code snippets allow you to execute these GraphQL operations more easily with the help of the Amplify JavaScript library. Choose the following options:
```

File: src/fragments/start/getting-started/vanillajs/setup.mdx [hunk 2]
```diff
@@ -159,35 +159,33 @@ amplify init
...
+- It creates a file called `aws-exports.js` in the `src` directory that holds all the configuration for the services you create with Amplify. This is how the Amplify JavaScript client library is able to get the necessary information to connect to your backend services.
 - It modifies the `.gitignore` file, adding some generated files to the ignore list
-- A cloud project is created for you in the AWS Amplify Console that can be accessed by running `amplify console`. The Console provides a list of backend environments, deep links to provisioned resources per Amplify category, status of recent deployments, and instructions on how to promote, clone, pull, and delete backend resources
+- An Amplify project is created for you that can be access within the AWS Console by running `amplify console`. The AWS Amplify console provides a list of backend environments, deep links to provisioned resources per Amplify category, deployment statuses, and instructions on how to promote, clone, pull, and delete backend resources.
```
Comment (josefaidt) [inline src/fragments/start/getting-started/vanillajs/setup.mdx:191]:
```suggestion
- An Amplify project is created for you that can be accessed within the AWS Console by running `amplify console`. The AWS Amplify console provides a list of backend environments, deep links to provisioned resources per Amplify category, deployment statuses, and instructions on how to promote, clone, pull, and delete backend resources.
```

File: src/fragments/start/getting-started/vanillajs/data-model.mdx [hunk 3]
```diff
@@ -40,32 +30,37 @@ The CLI should open this GraphQL schema in your text editor.
...
 
-A type decorated with the `@model` directive will scaffold out the database table for the type (Todo table), the schema for CRUD (create, read, update, delete) and list operations, and the GraphQL resolvers needed to make everything work together.
+A GraphQL type decorated with the `@model` directive will scaffold out the database table for the type (Todo table), the schema for CRUD (create, read, update, delete) and list operations, and the GraphQL resolvers needed to make everything work together.
 
 From the command line, press __enter__ to accept the schema and continue to the next steps.
 
 ## Deploy your GraphQL API
 
-Now that the API has been successfully created. We need to push our updated configuration to the cloud so our API can be deployed:
+Now that the API has been successfully created locally, we need to deploy the API to the cloud:
```
Comment (siegerts) [inline src/fragments/start/getting-started/vanillajs/data-model.mdx:55]:
```suggestion
Now that the API has been successfully created locally, you need to deploy the API to the cloud:
```

**Review by siegerts — CHANGES_REQUESTED** [review]

File: src/fragments/start/getting-started/vanillajs/setup.mdx [hunk 4]
```diff
@@ -159,35 +159,33 @@ amplify init
...
-
-Distribution directory path (dist)
-
-Build command (npm run-script build)
-
-Start command (npm run-script start)
-
-# This is the profile you created with the `amplify configure` command in the introduction step.
-Do you want to use an AWS profile
+? Enter a name for the project: amplifyjsapp
```
Comment (wlee221) [inline src/fragments/start/getting-started/vanillajs/setup.mdx:162]: What do you think about aligning answers with `#`, like you did with `data-model.mdx`? 

```suggestion
? Enter a name for the project:
# amplifyjsapp
```

Comment (wlee221) [reply src/fragments/start/getting-started/vanillajs/setup.mdx:162]: I don't have a strong preference between the two, but we should align on a common pattern.

Comment (rachelnabors) [reply src/fragments/start/getting-started/vanillajs/setup.mdx:162]: Not sure on this. I find it hard to scan "did I just get the same SLI output?" with the #'s.

**Review by siegerts — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**